pkg/cri/resource-manager/policy/policy.go

```diff
@@ -266,6 +268,13 @@ func (p *policy) UpdateResources(c cache.Container) error {
 	return p.backend.UpdateResources(c)
 }
 
+// ExportResourceData exports/updates resource data for the container.
+func (p *policy) ExportResourceData(c cache.Container, syntax DataSyntax) {
```

Comment:
What is this `syntax` here for?

Reply:
The original idea was to support exporting the same policy-specific data in different formats, for instance as JSON or as a /bin/sh-syntax source-able file. Only the latter has been implemented.

It'd be probably more reasonable to let the policy backend only supply raw data (as a map[string]string) for the container and do the actual formatting in the agnostic layer.

Reply:
I moved formatting out of the backends to policy/policy.go with the backends only providing raw data in a map. Got rid of DataSyntax and the related constants. We always export in shell syntax only now.